### PR TITLE
ISSUE_TEMPLATE: Use the newer issue templates

### PR DESCRIPTION
**Pull request by RedSoxFan**
This utilizes the newer Github issue templates. They allow for the user
to specify what type of issue they are submitting to allow for a more
specific issue template to be shown.

In addition to a hopefully easier to read/parse/follow bug report
template, this also include templates for enhancements and i3
compatibility. This also includes a link to the IRC under the section
title Questions.

For the three templates, the labels bug, enhancement, and i3-compat will
be automatically applied for the appropriate report to assist in
triaging.

Hopefully, this will result in less questions and issues for new window
management functionality on Github and allows for better quality issues
being submitted. At the very least, it allows us to outline our
stances for bugs, enhancements, and i3-compatibility in an easier to
read format.

The is what the user will see when they click on New Issue
screenshot-20200705-204317

When they click on one of the templates, they will see the corresponding template:
- <details>
   <summary>Bugs</summary>

   screenshot-20200705-203717
</details>

- <details>
   <summary>Enhancements</summary>

   screenshot-20200705-203648
</details>

- <details>
   <summary>i3 Compatibility</summary>

   screenshot-20200705-203615
</details>

- Questions links to http://webchat.freenode.net/?channels=sway&uio=d4 just like the README

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+name: Bugs
+about: Crashes and other bugs
+title: ''
+labels: 'bug'
```
Comment (emersion) [inline .github/ISSUE_TEMPLATE/bug_report.md:5]: Up until now I've only labelled issues as "bugs" until the bug report was complete (debug logs, enough information to start debugging). I think it's a good idea to have a label for all bugs, but I wonder if we should have a replacement for the old scheme. Maybe I can just add the "waiting on reporter" label when typing out the message asking for missing info.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+assignees: ''
```
Comment (emersion) [inline .github/ISSUE_TEMPLATE/bug_report.md:6]: Nit: can we omit `assignees`?

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Questions
+    url: "http://webchat.freenode.net/?channels=sway&uio=d4"
+    about: "Please ask questions on IRC at #sway on irc.freenode.net"
```
Comment (emersion) [inline .github/ISSUE_TEMPLATE/config.yml:5]: Nit: I feel like "in #sway" sounds better, but I may be wrong

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (RedSoxFan) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Oh looks like you can omit them. Removed.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (RedSoxFan) [reply .github/ISSUE_TEMPLATE/bug_report.md:4]: My vote would be to use `waiting on reporter` for that. I don't think `bug` is the correct label to use for that.

Comment (emersion) [reply .github/ISSUE_TEMPLATE/bug_report.md:4]: Works for me!

**Pull request merged**